test/noise.test.ts

```diff
@@ -3,8 +3,8 @@ import DuplexPair from 'it-pair/duplex';
 import {Noise} from "../src";
 import {createPeerIdsFromFixtures} from "./fixtures/peer";
 import Wrap from "it-pb-rpc";
-import {random} from "bcrypto";
 import sinon from "sinon";
+import crypto from 'crypto';
```

Comment:
You didn't list this dependency.
Also you have this method in libp2p-crypto
https://github.com/libp2p/js-libp2p-crypto#cryptorandomBytesnumber